coreneuron/CMakeLists.txt

```diff
@@ -42,6 +46,9 @@ file(GLOB_RECURSE scopmath_c_files "scopmath_core/*.c*")
 
 # set mod2c binary and files from built in
 if(NOT EXISTS ${MOD2C})
+    if(${ENABLE_ISPC_TARGET})
+        message(FATAL_ERROR "To use ispc target, you must ")
```

Comment:
Do you really want to `FATAL_ERROR` ? Maybe more explanation in the error message?

Reply:
I think it should be a failing error, but the message was nonsensical of course...